// fail

fn main() = {
    x @ while true {
        let c = async {
            break at x.
        }.
    }.
}.